.gitignore

```diff
@@ -14,3 +14,4 @@ Maple2Storage/obj/
 MaplePacketLib2/obj/
 MapleServer2/obj/
 MapleServer2/Properties/launchSettings.json
+/Maple2Storage/Resources
```

Comment:
Can you remove the slash at the beginning?

Reply:
Done